add seeds script

anysite opened the pull request:
Added 100 new comments, users, and items, just run it from inside backend machine

docker exec -it anythink-backend  node backend/scripts/seeds.js

Commit:
add seeds script
Added 100 new comments, users, and items, just run it from inside backend machine

docker exec -it anythink-backend  node backend/scripts/seeds.js

vanessa-cooper reviewed (changes requested):
Umm, something is off. Make sure all checks pass and only then try and merge the PR.

Commit:
add seeds script
Added 100 new comments, users, and items, just run it from inside backend machine

docker exec -it anythink-backend  node backend/scripts/seeds.js

vanessa-cooper reviewed (approved):
Nice! Looks like there are some records in your local DB. Now you can *merge the PR* and come *back to Snack* once you’re ready.
